src/main/java/me/dinowernli/grpc/polyglot/command/Method.java

```diff
@@ -0,0 +1,20 @@
+package me.dinowernli.grpc.polyglot.command;
+
+public class Method {
```

Comment:
Do we really need to define these custom types (here and services below)?

Can we not just use the methoddescriptor produced by protobuf?